src/utils/string.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * ex:  nguyễn hoài danh => nguyen-hoai-danh
+ * @param text
+ * @returns
+ */
+export function convertToSlug(text: string) {
+  return text
+    .toLowerCase()
+    .replace(/ /g, '-')
+    .replace(/[^\w-]+/g, '')
```

Comment:
Should we add `.` to `/[^\w-.]+/`? This support tokens with `.` in their symbol

Reply:
got it.